src/styles/index.scss

```diff
@@ -1,3 +1,253 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+}
+
+a:visited {
+  color: inherit;
+  text-decoration: none;
+}
+
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: rgba(0, 172, 220, 1);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  transition: 300ms;
+
+  &:hover {
+    background-color: rgba(255, 255, 255, 1);
+    color: rgba(0, 172, 220, 1);
+    border: 1px solid rgba(0, 172, 220, 1);
+  }
+}
```

Comment:
The `.buy` class has a hover effect defined. Ensure that this effect is consistent with the design requirements and that the color contrast is accessible.